adafruit_magtag/peripherals.py

```diff
@@ -71,7 +76,9 @@ def __init__(self):
             switch.pull = Pull.UP
             self.buttons.append(switch)
 
-    def play_tone(self, frequency, duration):
+    def play_tone(
+        self, frequency: Union[float, int], duration: Union[float, int]
```

Comment:
`float` also includes `int` so no need to use `Union` - `float` is sufficient!

Reply:
Oh, cool! Thanks again for your help and for kindly teaching me!